docs/organizations/managing-security-and-risk.md

```diff
@@ -445,3 +445,17 @@ Each Codacy issue reported by Security and risk management belongs to one of the
 | **Visibility**                     | Logging should always be included for security events to better allow attack detection and help defend against vulnerabilities.                                                                                  |
 | **XSS**                            | Cross-Site Scripting (XSS) attacks inject malicious client-side scripts into trusted websites that are visited by the end users.                                                                                 |
 | **Other**                          | Other language-specific security issues.                                                                                                                                                                         |
+
+## Scan types
```

Comment:
I'm not sure if we are being consistent across the UI regarding the order of the different filters/dimensions (severity, status, security category, scan type)...
If we are, consider ordering the corresponding sub-sections following the same order, and moving [Languages checked for security issues](https://github.com/codacy/docs/pull/2128/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R220) to the end (as we don't have a filter for it).

Reply:
Good point